src/main/resources/templates/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
```

Comment:
This file is getting unruly with how much code is in it.  At some point it might be worth breaking it apart.